### PR TITLE
Remove boilerplate word

### PR DESCRIPTION
**Pull request by npentrel**
This has been bothering me for a while. Our sample code is not actually boilerplate code ([wiki definition](https://en.wikipedia.org/wiki/Boilerplate_code)).

### CODE REVIEW

File: docs/registry/advanced/create-subtype.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The following steps guide you through this process in more detail:
      - [Example component in Python](https://github.com/viamrobotics/viam-python-sdk/blob/main/examples/complex_module/src/gizmo/api.py)
      - [Example service in Python](https://github.com/viam-labs/speech-service-api/blob/main/src/speech_service_api/api.py)
 
-1. In the root directory of your module, you need to generate some boilerplate files.
+1. In the root directory of your module, you need to generate some starter files.
```
Comment (JessamyT) [inline docs/registry/advanced/create-subtype.md:60]: Is there another word for code that is the same in every project?

**Review by JessamyT — APPROVED** [review]
LGTM unless we can find a word other than “starter,” which in my mind could possibly connote something that only beginners use instead of something that is necessary but the same in every project.

File: docs/registry/advanced/create-subtype.md (hunk 1 above)
Comment (npentrel) [reply docs/registry/advanced/create-subtype.md:60]: template?

Comment (npentrel) [reply docs/registry/advanced/create-subtype.md:60]: I think in fact these are just config files
```suggestion
1. In the root directory of your module, you need to generate some configuration files.
```

File: docs/tutorials/control/flutter-app.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ This tutorial assumes you already have a machine [configured](/configure/) on th
 
 ## Set up your Flutter development environment
 
-This tutorial uses [Visual Studio Code](https://code.visualstudio.com/download) (VS Code) as the development environment (IDE), and uses the VS Code [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) to generate necessary boilerplate code.
+This tutorial uses [Visual Studio Code](https://code.visualstudio.com/download) (VS Code) as the development environment (IDE), and uses the VS Code [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) to generate necessary starter code.
```
Comment (npentrel) [inline docs/tutorials/control/flutter-app.md:50]:
```suggestion
This tutorial uses [Visual Studio Code](https://code.visualstudio.com/download) (VS Code) as the development environment (IDE), and uses the VS Code [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) to generate necessary project code.
```

File: docs/tutorials/control/flutter-app.md [hunk 3]
```diff
@@ -94,10 +94,10 @@ You can run `fvm use 3.19.6` in the terminal before building your sample app to
 
 {{<imgproc src="/tutorials/flutter-app/project-name.png" resize="1000x" declared-dimensions="true" alt="Entering the name for the project in VS Code: 'smart_machine_app'.">}}
 
-When you hit **Enter**, Flutter auto-generates a project folder with lots of useful boilerplate code.
+When you hit **Enter**, Flutter auto-generates a project folder with lots of useful starter code.
```
Comment (npentrel) [inline docs/tutorials/control/flutter-app.md:97]:
```suggestion
When you hit **Enter**, Flutter auto-generates a project folder with a useful starter project.
```

**Pull request merged**